terraso_backend/apps/project_management/models/sites.py

```diff
@@ -14,15 +14,32 @@
 # along with this program. If not, see https://www.gnu.org/licenses/.
 from django.db import models
 
+from apps.core import permission_rules
 from apps.core.models.commons import SlugModel
 
+from .projects import Project
+
 
 class Site(SlugModel):
     class Meta(SlugModel.Meta):
         abstract = False
 
+        rules_permissions = {"change": permission_rules.allowed_to_edit_site}
+
     name = models.CharField(max_length=200)
+    name = models.CharField(max_length=200)
+    latitude = models.FloatField()
     latitude = models.FloatField()
     longitude = models.FloatField()
+    longitude = models.FloatField()
 
     field_to_slug = "id"
+    field_to_slug = "id"
```

Comment:
Question: If the slug is not based on a human redable name, do we need to use SlugModel?